presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeoFunctions.java

```diff
@@ -997,6 +1017,7 @@ public static Slice stExteriorRing(@SqlType(GEOMETRY_TYPE_NAME) Slice input)
     @SqlType(GEOMETRY_TYPE_NAME)
     public static Slice stIntersection(@SqlType(GEOMETRY_TYPE_NAME) Slice left, @SqlType(GEOMETRY_TYPE_NAME) Slice right)
     {
+
```

Comment:
Unnecessary change.

Reply:
Removed. This is causing check-style fail.